tests/wpt/meta/preload/preload-csp.sub.html.ini

```diff
@@ -0,0 +1,3 @@
+[preload-csp.sub.html]
```

Comment:
It's strange to me that the `font-src` CSP tests pass, but this one doesn't. I double-checked if the request has all the correct information and added various `println!` statements in `rust-content-security-policy`, but wasn't able to figure out what we are still missing in `create_link_request`.

Reply:
It's because we don't support `track` requests yet. All the other requests are correctly blocked.